deploy/templates/notificationtemplates/sandbox/userdeactivated/notification.html

```diff
@@ -41,23 +41,21 @@
 >
 
     <p>
-        You are receiving this email because you have a Developer Sandbox for Red Hat OpenShift
-        account associated with {{.UserEmail}}.
+        You are receiving this email because the Developer Sandbox account associated with {{.UserEmail}} is now deactivated.
+        All your data on Developer Sandbox has been deleted. You can start a new trial at {{.RegistrationURL}}, whenever you would like.
     </p>
 
     <p>
-        Your account is now deactivated and all your data on Developer Sandbox for Red Hat OpenShift has been deleted.
-        You can request new access by signing up again at {{.RegistrationURL}}
+        To learn more about how Red hat products, support and training could be benefit your organization <a href="https://www.redhat.com/en/contact">request a callback.</a>
```

Comment:
This sentence looks a bit weird to me, I'm not a native speaker though. 
Shouldn't it be:
```suggestion
        To learn more about how Red Hat products, support, and training could benefit your organization, <a href="https://www.redhat.com/en/contact">request a callback.</a>
```
Things to fix:
* capitalization of `Hat`
* drop `be` (or should it rather contain `... be beneficial for your organization` instead?)
* comma after `support`
* comma after `organization`